pkg/operator/controller/controller_test.go

```diff
@@ -285,14 +288,10 @@ var _ = Describe("Controller", func() {
 				doReconcile(args)
 				Expect(setDeploymentsReady(args)).To(BeTrue())
 
-				runbookURLTemplate := getRunbookURLTemplate()
+				runbookURLTemplate := alerts.GetRunbookURLTemplate()
 
-				rule := &promv1.PrometheusRule{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      "prometheus-cdi-rules",
-						Namespace: cdiNamespace,
-					},
-				}
+				rule, err := rules.BuildPrometheusRule(cdiNamespace)
```

Comment:
So this PrometheusRule is defined solely to call a GET, do we need the entire rule spec?

Reply:
not sure I understand what you mean here sorry

Reply:
Fixed